backend/docker/prod/lambda/Dockerfile

```diff
@@ -14,17 +17,22 @@ RUN yum -y install openssl-devel bzip2-devel libffi-devel postgresql-devel gcc u
 RUN amazon-linux-extras install $PYTHON_VERSION
 RUN yum -y install python38-devel
 
+RUN groupadd -r ${CONTAINER_USER_GROUP} && useradd -m -r -g ${CONTAINER_USER_GROUP} ${CONTAINER_USER}
+RUN chown -R ${CONTAINER_USER}:root /usr/local
+
+USER ${CONTAINER_USER}
+
 ## Add your source
 WORKDIR ${FUNCTION_DIR}
 
-COPY backend/requirements.txt ./requirements.txt
+COPY --chown=${CONTAINER_USER}:root backend/requirements.txt ./requirements.txt
 RUN $PYTHON_VERSION -m pip install -U pip
 RUN $PYTHON_VERSION -m pip install -r requirements.txt -t .
```

Comment:
when I deployed, the build for this command failed when building the image.
I think the issue is related to `FUNCTION_DIR` pointing to /home/app when it should point to the home folder of the new user you created

Reply:
Resolved by adding permissions to app dir @dbalintx

Reply:
Updated the permissions for FUNCTION_DIR